README.md

```diff
@@ -119,11 +117,9 @@ This recipe performs a synchronous detection and install of available Windows up
 ### Attributes
 The following attributes are used to configure the `wsus-client::update` recipe, accessible via `node['wsus_client'][attribute]`.
 
-Attribute      | Description                                                          | Type   | Default
----------------|----------------------------------------------------------------------|--------|--------
-update_timeout | The maximum amount of time the synchronous update is allowed to run. | FixNum | `7200`
-
-> **Note:** mixlib-shellout < 2.0.1 does not terminating the process after the allowed timeout. Chef 12.0.4 includes the new version.
+Attribute     | Description                                                              | Type                  | Default
+--------------|--------------------------------------------------------------------------|-----------------------|--------
+download_only | Determines whether available update should be installed once downloaded. | TrueClass, FalseClass | `false`
```

Comment:
The 'download_only' attribute doesn't seem to be set above. Or am I missing something ?